recipe/meta.yaml

```diff
@@ -45,6 +45,8 @@ test:
     - curl https://raw.githubusercontent.com/conda-forge/curl-feedstock/master/LICENSE
     - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
     - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+    - nm $PREFIX/lib/libcurl.so | grep libssh2_scp_recv2  # [not win]
```

Comment:
Also while `nm` does exist. `.so` is really only on Linux (except for some Python extensions). Should be `$SHLIB_EXT` instead.